frontend/translations/en.ts

```diff
@@ -1,6 +1,8 @@
 // All guidelines for the translations can be found in the file README.md
 export const en = {
   'app.api-testing.label': 'API testing',
+  'api.back_to_reports_list.label': '< Back to Report List',
```

Comment:
is `api` correct at the begging or it should be `app` as everywhere else

Reply:
hahah app of course :D